[2025-06-23T00:12:35.679+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-23T00:12:35.716+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-06-22T00:00:00+00:00 [queued]>
[2025-06-23T00:12:35.731+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-06-22T00:00:00+00:00 [queued]>
[2025-06-23T00:12:35.734+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-23T00:12:35.769+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform> on 2025-06-22 00:00:00+00:00
[2025-06-23T00:12:35.778+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'transform', 'scheduled__2025-06-22T00:00:00+00:00', '--job-id', '148', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpqmgj0tzk']
[2025-06-23T00:12:35.780+0000] {standard_task_runner.py:91} INFO - Job 148: Subtask transform
[2025-06-23T00:12:35.783+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=39) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-06-23T00:12:35.784+0000] {standard_task_runner.py:63} INFO - Started process 48 to run task
[2025-06-23T00:12:35.849+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline.transform scheduled__2025-06-22T00:00:00+00:00 [running]> on host 2c64c5599160
[2025-06-23T00:12:35.964+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-22T00:00:00+00:00'
[2025-06-23T00:12:35.965+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-23T00:12:53.284+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-06-23T00:12:53.285+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-23T00:12:53.299+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=transform, run_id=scheduled__2025-06-22T00:00:00+00:00, execution_date=20250622T000000, start_date=20250623T001235, end_date=20250623T001253
[2025-06-23T00:12:53.368+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-23T00:12:53.425+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-23T00:12:53.441+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
